doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -1747,170 +2230,256 @@ The following table indicates the software maturity levels of the support for Go
 
    .. _software_maturity_fast_pair_use_case:
 
-   .. list-table:: Google Fast Pair use case support
-      :header-rows: 1
-      :align: center
-      :widths: auto
-
-      * - Use case
-        - |NCS| sample demonstration
-        - nRF52810
-        - nRF52811
-        - nRF52820
-        - nRF52832
-        - nRF52833
-        - nRF52840
-        - nRF5340
-        - nRF54H20
-        - nRF54L05
-        - nRF54L10
-        - nRF54L15
-        - nRF9131
-        - nRF9151
-        - nRF9160
-        - nRF9161
-      * - **Input device**
-        - :ref:`fast_pair_input_device`
-        - --
-        - --
-        - --
-        - Experimental
-        - Experimental
-        - Experimental
-        - Experimental
-        - Experimental
-        - Experimental
-        - Experimental
-        - Experimental
-        - --
-        - --
-        - --
-        - --
-      * - **Locator tag**
-        - :ref:`fast_pair_locator_tag`
-        - --
-        - --
-        - --
-        - Supported
-        - Supported
-        - Supported
-        - Experimental
-        - Experimental
-        - Experimental
-        - Experimental
-        - Supported
-        - --
-        - --
-        - --
-        - --
+   .. tabs::
+
+      .. tab:: nRF52 Series
+
+         .. list-table:: Google Fast Pair use case support
+            :header-rows: 1
+            :align: center
```

Comment:
I think if  :align: center is removed from Google Fast Pair use case support and Google Fast Pair feature support, then the tables will be left aligned like in other features to be consistent.

![image](https://github.com/user-attachments/assets/c48a68bc-ac25-4bb5-9735-21594a820ecf)